lighthouse-plugin-publisher-ads/audits/ad-request-from-page-start.js

```diff
@@ -18,8 +18,8 @@ const {auditNotApplicable, runWarning} = require('../messages/common-strings');
 const {Audit} = require('lighthouse');
 
 const UIStrings = {
-  title: 'Latency of first ad request',
-  failureTitle: 'Reduce latency of first ad request',
+  title: 'First ad request time',
+  failureTitle: 'Reduce time to send the first ad request',
```

Comment:
This seems a little abbreviated/ambiguous. Maybe "Make the first ad request earlier", "Reduce time before first ad request is sent", etc.?

Reply:
I should clarify: I think the green text is a vast improvement over the red text. Just think it could still be improved a bit further.